docs/bin/create-image.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const puppeteer = require('puppeteer'); // eslint-disable-line
+const path = require('path'); // eslint-disable-line
+const pack = require('../../package.json'); // eslint-disable-line
+
+const fileName = process.argv[2];
+const token = process.argv[3] || process.env.MAPBOX_ACCESS_TOKEN;
```

Comment:
Ah sorry, I meant that it would be good to check for `MAPBOX_ACCESS_TOKEN` in addition to `MapboxAccessToken` since people may have it saved either way. That's what the benchmark tests do as seen [here](https://github.com/mapbox/mapbox-gl-js/blob/master/bench/lib/access_token.js#L2-L3).